import-scripts/update-az-mskimpact.sh

```diff
@@ -62,7 +61,7 @@ function transfer_to_az_sftp_server() {
     # Connect and transfer data
     # With use of here-doc, there must be no leading whitespace until EOF
     sftp -i "$TRANSFER_KEY" "$SFTP_USER"@"$SERVICE_ENDPOINT" -b <<EOF
-put -R "$AZ_DATA_HOME/az_mskimpact" "$AZ_REPO_NAME"
+put -R "$AZ_DATA_HOME/$AZ_MSKIMPACT_STABLE_ID" "$AZ_REPO_NAME"
```

Comment:
you have `AZ_MSK_IMPACT_DATA_HOME` defined in automation_environment

Reply:
good catch, just fixed